renku/core/commands/providers/zenodo.py

```diff
@@ -236,7 +236,7 @@ def get_jsonld(self):
 
     def get_files(self):
         """Get Zenodo files metadata as ``ZenodoFile``."""
-        if len(self.files) == 0:
+        if not self.files:
             raise LookupError('no files have been found')
```

Comment:
It's better to mention that there might be protected/restricted files: `'no files have been found or there are files with restricted access'`

Reply:
Sure. Maybe the following wording would work better?

`no files have been found - deposit empty or protected'

Reply:
`no files have been found - deposit is empty or protected` sounds better to me but up to you.